units/src/amount/signed.rs

```diff
@@ -144,7 +176,7 @@ impl SignedAmount {
     ///
     /// ```
     /// # use bitcoin_units::amount::{SignedAmount, Denomination};
-    /// let amount = SignedAmount::from_sat(100_000);
+    /// let amount = SignedAmount::from_sat_unchecked(100_000);
     /// assert_eq!(amount.to_btc(), amount.to_float_in(Denomination::Bitcoin))
```

Comment:
Ouch, `git blame` shows that I wrote these examples only a month ago. This is a terrible example, it does now show _why_ one would use the function and even worse the result of the to calls in the assertion are opaque. I'll fix it as a separate PR.